src/dynsight/_internal/data_processing/spatial_smoothing.py

```diff
@@ -62,9 +64,10 @@ def spatial_smoothing(
 
     shape = array.shape
     dtype = array.dtype
-
     shared_array = Array(ctypes.c_double, array.size, lock=False)
-    shared_array_np = np.frombuffer(shared_array, dtype=dtype).reshape(shape)
+    # The mypy error [call-overload] is ignored as it is considered.
```

Comment:
Remove the full stop at the end.

Also, a bit more detail is needed: `np.frombuffer type is 'buffer-like', which contains 'Array'`